src/oca_github_bot/webhooks/on_pr_update_check_cla.py

```diff
@@ -0,0 +1,384 @@
+# Copyright (c) Camptocamp 2018
+# Copyright (c) Akretion 2014
+
+import logging
+from ..router import router
+# import aiosqlite
+import sqlite3
+import json
+import re
+
+import requests  # FIXME remove
+from requests.auth import HTTPBasicAuth  # FIXME remove
+
+from .. import config
+from ..odoo_client import login
+
+_logger = logging.getLogger(__name__)
+RE_WEBLATE_COMMIT_MSG = r'''^Translated using Weblate.*
+Translate-URL: https://translation\.odoo-community\.org/'''
+
+
+@router.register("pull_request")
+async def on_pr_check_cla(event, gh, *args, **kwargs):
+    checker = CLACheck(event, gh)
+    return checker.check_cla()
```

Comment:
@gurneyalex Thanks! 

Could you move the bulk of the CLA bot to the tasks directory and decorate it with `@task()`, and here call it with `delay()`. That's the easiest to queue it, and avoid the need to asyncify everything.

Reply:
thanks @sbidoul I'll do this

Reply:
done, still needs testing (I have not run this yet...)